src/main/java/appeng/core/features/registries/grinder/GrinderRecipeManager.java

```diff
@@ -393,4 +399,123 @@ else if( this.item != other.item )
 		}
 
 	}
+
+	/**
+	 * Internal {@link IInscriberRecipeBuilder} implementation.
+	 * Needs to be adapted to represent a correct {@link IInscriberRecipe}
+	 */
+	private static final class Builder implements IGrinderRecipeBuilder
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a constructor to the class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1258)